cfme/tests/configure/test_access_control.py

```diff
@@ -578,11 +586,9 @@ def _test_vm_removal():
 @pytest.mark.tier(3)
 @pytest.mark.parametrize(
     'product_features, action',
-    [({version.LOWEST: [
-        ['Everything', 'Compute', 'Infrastructure', 'Virtual Machines', 'Accordions'],
-        ['Everything', 'Access Rules for all Virtual Machines', 'VM Access Rules', 'Modify',
-         'Provision VMs']], },
-      _test_vm_provision)])
+    [([['Everything', 'Access Rules for all Virtual Machines', 'VM Access Rules', 'View'],
+       ['Everything', 'Compute', 'Infrastructure', 'Virtual Machines', 'Accordions']],
+      _test_vm_view)])
```

Comment:
Since this action is the same method in all cases, why parametrize it? Can't you call it directly instead of this obfuscated `action`?

Reply:
Updated to use directly function instead of parametrization